src/api/docker.ts

```diff
@@ -11,7 +11,26 @@ import { platform } from 'os';
 import * as commandExists from 'command-exists';
 import { demuxStream } from './utils';
 
-export const docker = new dockerode();
+export function getDockerOptions(): dockerode.DockerOptions {
+  const dockerOptions: dockerode.DockerOptions = {};
+
+  if (process.env.DOCKER_HOST) {
+    // parse docker URI in environment variable DOCKER_HOST="tcp://0.0.0.0:2375"
+    const re = /tcp\:\/\/([\w|\.]+)(\:?(\d+))/gi;
+    const [ , host, , port ] = re.exec(process.env.DOCKER_HOST)
+    if (host) {
+      dockerOptions.host = host;
+    }
+    if (port) {
+      dockerOptions.port = port;
+    }
+  }
+
+  return dockerOptions
+}
+
+export const docker = new dockerode(getDockerOptions());
+
 const binds = platform() === 'darwin' ? [] : ['/var/run/docker.sock:/var/run/docker.sock'];
```

Comment:
@jkuri i didnt fully understand what ``binds`` does, but i assume binds should be different when using docker via tcp